package.json

```diff
@@ -9,18 +9,48 @@
   "license": "SEE LICENSE IN LICENSE",
   "scripts": {
     "argos": "argos upload test/visual/screenshots/chrome --token $ARGOS_TOKEN || true",
-    "build": "rimraf packages/**/styles/**/*.js && yarn build-styling",
-    "build-styling": "node packages/sass-render/bin/sass-render.js -s \"packages/*/scss/**/*.scss\"",
+    "dev": "wireit",
     "deploy-storybook": "yarn --cwd packages/storybook storybook-to-ghpages",
     "dist": "yarn build && lerna run dist",
     "lint": "npm-run-all --parallel lint:*",
     "lint:js": "eslint *.js packages --ignore-path .gitignore",
     "lint:scss": "stylelint packages/**/*.scss",
-    "storybook": "yarn --cwd packages/storybook start-storybook -p 6006 --modern",
+    "storybook": "wireit",
+    "styles:build": "wireit",
     "test": "karma start",
     "test:visual": "rimraf tmp && BABEL_ENV=visual webpack --config test/visual/webpack.config.js && rimraf test/visual/screenshots/chrome/* && vrtest run --config test/visual/vrtest.config.js --record",
     "watch": "node scripts/watcher.js"
   },
+  "wireit": {
+    "dev": {
+      "dependencies": [
+        "storybook"
+      ]
+    },
+    "storybook": {
+      "command": "yarn --cwd packages/storybook start-storybook -p 6006 --modern",
+      "dependencies": [
+        "styles:watch"
+      ],
+      "service": true
+    },
+    "styles:build": {
+      "command": "node packages/sass-render/bin/sass-render.js -s \"packages/*/scss/**/*.scss\"",
+      "dependencies": [
+        "styles:clean"
+      ],
+      "files": [
+        "packages/*/scss/**/*.scss"
+      ]
+    },
+    "styles:clean": {
+      "command": "rimraf packages/**/styles/**/*.js"
+    },
+    "styles:watch": {
+      "command": "yarn run styles:build --watch",
+      "service": true
+    }
```

Comment:
Because wireit watcher doesn't have SASS-specific intelligence, this seems to render all SCSS files on any one change?

Reply:
I do see that. It looks like an issue that they are working to address: https://github.com/google/wireit/issues/168

Reply:
To me it's a pretty clear blocker until they solve it upstream.

For our partial watching problem, we could explore how to wire up native `sass -w` instead, similar to how `packages/institute-theme` does it. Wireit may or may not be a part of the solution, maybe it's still useful without its watcher function.

Reply:
From what I gather, there's no way to determine that a change has been made to an imported file.

For example:

A change is made to `packages\cxl-ui\scss\cxl-app-layout\_layout.scss`. Run `sass` on `packages\cxl-ui\scss\cxl-app-layout.scss`.

The reason why `build` and `build-styling` work is because _we do_ run `sass-render` on _everything_. Even if `scripts/watcher.js` is able to detect a change in `packages\cxl-ui\scss\cxl-app-layout\_layout.scss`, it won't update `packages\cxl-ui\src\styles\cxl-app-layout-css.js`.

Herein lies the problem. Storybook won't update unless `packages\cxl-ui\src\styles\cxl-app-layout-css.js` is updated.

The only two solutions I see are:

1) Get rid of partials
2) Run `sass-render` on all files whenever any `scss` file is changed (which is pretty much what we do now manually).